bin/verify-changelog-updated.js

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env node
+// Load our dependencies
+var assert = require('assert');
+var fs = require('fs');
+
+// Load in our CHANGELOG
+var content = fs.readFileSync('CHANGELOG.md');
+
+// Verify we have a tag to find
+var version = process.env.FOUNDRY_VERSION;
+assert(version, 'Expected `env.FOUNDRY_VERSION` to be defined but it was not');
+
+// Verify our tag is present in the CHANGELOG
+if (content.indexOf(version) === -1) {
```

Comment:
It seems like a far-out edge case (pretty sure after the first few releases, people will get used to it). That being said, we can't guarantee the `x.y.z -` format (we might have a major release with multiple bullets) but `\nx.y.z` seems reasonable/easy to upgrade (e.g. `\n[^0-9]+x.y.z` to handle a scenario where we might use boldening or `##`)